mks_server.py

```diff
@@ -212,12 +226,20 @@ def update_readings(self):
                         data2 = 0
                     else:
                         data2 = float(rdg2)
-                except ValueError:
+                except ValueError as e:
+                    logging.debug('Serial reading 2 error: {}'.format(e))
+                    error = True
                     data2 = float('NaN')
+                if error:
+                    logging.debug('Error occured. Values:')
+                    logging.debug('    Pressure: {}'.format(result.pressure))
+                    logging.debug('    Full Scale: {}'.format(result.full_scale))
```

Comment:
It's better to create one multiline string and then make one logging call. If you make separate calls it's much harder to piece the log together because these statements may get interleaved with other messages, especially when using syslog.